.circleci/config.yml

```diff
@@ -69,7 +69,7 @@ jobs:
               ram=$(awk -F"[: ]+" '/MemTotal/ {print $2;exit}' /proc/meminfo)
               os=$(uname -sr)
               printf "{\n\"CircleCI\": {\n\"arch\": \"${arch}\",\n\"cpu\": \"${cpu}\",\n\"machine\": \"CircleCI\",\n\"os\": \"${os}\",\n\"ram\": \"${ram}\"\n},\n\"version\": 1\n}" | tee ~/.asv-machine.json
-              python -u runtests.py --bench
+              python -u runtests.py -q --bench
```

Comment:
For reference, it seems that locally I had to do this for the syntax to be accepted:

`python -u runtests.py --bench [-q]`

I didn't confirm that the flag actually propagates through, but doesn't raise any errors and starts trying to build.